Aptoide/aptoide-client-v8
# [Monetization/ASV-1140] Add ab test placements

**What does this PR do?**

   Exactly the same as https://github.com/Aptoide/aptoide-client-v8/pull/755 , but adds placements for AB tests instead.

**Database changed?**

   No


**How should this be manually tested?**

  See [this](https://github.com/Aptoide/aptoide-client-v8/pull/755) PR. It's the same, just test if it still works for appnext ads.

**What are the relevant tickets?**

  [ASV-1140](https://aptoide.atlassian.net/browse/ASV-1140)

**Code Review Checklist**
